python/polycap.pyx

```diff
@@ -121,6 +121,17 @@ def __send_google_analytics_launch_event():
 
 threading.Thread(target=__send_google_analytics_launch_event).start()
 
+# Some code from h5py/_errors.pyx to retain HDF5 functionality across multiple versions
```

Comment:
Copying that code from h5py here verbatim is not going to work, as it needs to be translated into C and add [here](https://github.com/PieterTack/polycap/blob/master/src/polycap-transmission-efficiencies.c#L81-L155).

Reply:
Yup, was reaching the same conclusion :D